bin/live/pycbc_live_collate_triggers

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Arthur Tolley
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+
+"""Find trigger files and combine them into a single hdf trigger merge file."""
+
+import glob
+import numpy
+import argparse
+import itertools
+import h5py
+import os
+import logging
+from datetime import datetime, timedelta
+
+from ligo.segments import segmentlist, segment
+
+import pycbc
+from pycbc.io import live as liveio
+from pycbc.events import cuts, veto
+
+# Set up the command line argument parser
+parser = argparse.ArgumentParser(description=__doc__)
+pycbc.add_common_pycbc_options(parser)
+liveio.add_live_trigger_selection_options(parser)
+cuts.insert_cuts_option_group(parser)
+
+parser.add_argument(
+    '--ifos',
+    nargs='+',
+    required=True,
+    help="The list of detectors to include triggers in the merged file"
+)
+
+# Option to output list of trigger files
+parser.add_argument(
+    '--output-trigger-file-list',
+    help='Name of the output file to save the list of trigger files.'
+)
+
+parser.add_argument(
+    '--output-file',
+    required=True,
+    help='The output file containing merged triggers.'
+)
+parser.add_argument(
+    "--bank-file",
+    required=True,
+    help="The bank file used in the search"
+)
+
+args = parser.parse_args()
+
+pycbc.init_logging(args.verbose)
+
+logging.info("Finding trigger files")
+trigger_files = liveio.find_trigger_files_from_cli(args)
+logging.info("%s files found", len(trigger_files))
+
+if args.output_trigger_file_list:
+    logging.info(
+        'Writing list of trigger files to %s ',
+        args.output_trigger_file_list
+    )
+    with open(args.output_trigger_file_list, 'w') as f:
+        for item in trigger_files:
+            f.write("%s\n" % item)
```

Comment:
@ArthurTolley is this still wanted or is it for testing only?

Reply:
It was good for testing because the file system does spend a lot of time looking for the files themselves depending on how many directories were being searched but I don't think we need it anymore.